pkg/apis/kops/cluster.go

```diff
@@ -149,6 +149,8 @@ type ClusterSpec struct {
 
 	// IAM field adds control over the IAM security policies applied to resources
 	IAM *IAMSpec `json:"iam,omitempty"`
+	// KubeEtcdEncryptionConfig holds the encryption config
+	KubeEtcdEncryptionConfig *KubeEtcdEncryptionConfig `json:"kubeEtcdEncryptionConfig,omitempty"`
```

Comment:
same at the above .. i think we should drop the etcd reference

Reply:
assuming we should the cli options; do we require additional validation on the contents of the config? .. should we check k8s version? .. etc

Reply:
We need to validate k8s version. We already have encryption options in etcd, we need to figure out an API name:

1. Does not have etcd, as we may be using another data source in the future
2. Kube is kinda redundant, this is a kubernetes kops manifest ;)

DataAtRestEncryption?

I am bad at naming, what do other people think?